keyboards/gray_studio/space65/keymaps/billiams/readme.md

```diff
@@ -0,0 +1,81 @@
+## Billiam's Space65 layout (with split backspace)
+
+This layout is optimized for vim users on MacOS with a split backspace.
+
+Settings:
+
+* The `CAPS LOCK` key is a function key.
+* The `ALT` and `CMD` keys are swapped to replicate the Mac layout.
+* RESET is available as `Fn`+ `Right Ctrl` + `ESC`
+* Underglow toggle is available as `Fn` + `Q`.
+* Backlighting toggle is available as `Fn` + `Z`
+* vim-style arrow key bindings H J K L in layer 1
+
+### Initial Installation
+
+I found the instructions to be longer than they had to be, and I ended up having to Google some steps anyway. These are the steps I took to get my keyboard setup, in case you are new to the process.
+
+1. Fork and Clone the qmk_firmware repo locally
+```
+# Choose one:
+git clone git@github.com:qmk/qmk_firmware.git # OR
+git clone https://github.com/qmk/qmk_firmware.git
+```
+2. Customize your layout by starting with a [keymap](https://github.com/qmk/qmk_firmware/tree/master/keyboards/gray_studio/space65/keymaps). I copied  the default and changed it to my liking.
+3. Before plugging in your keyboard into your computer, hold `SPACE` and `B` keys down
+4. Holding those keys down, plug the keyboard into your computer, which will put the keyboard in bootlegger mode. Note that if you've done this before, it could be that you need to simply press `ESC` prior to plugging in the board to put it in Bootlegger mode.
```

Comment:
bootloader* mode :smile: 

```suggestion
4. Holding those keys down, plug the keyboard into your computer, which will put the keyboard in bootloader mode. Note that if you've done this before, it could be that you need to simply press `ESC` prior to plugging in the board to put it in Bootloader mode.
```